packages/devtools-network-request/package.json

```diff
@@ -1,14 +0,0 @@
-{
-  "name": "devtools-network-request",
```

Comment:
`devtools-network-request` is currently also used by `devtools-source-map` for fetching mapping files. If you want to remove the package, we should also update `devtools-source-map` as well to stop using it.